exampleSite/config.toml

```diff
@@ -24,6 +24,8 @@ DefaultContentLanguage           = "en"                    # Default language fo
         localTime                = true                    # Show your current local time in contact section
         timeZone                 = "America/Los_Angeles"   # Your timezone as in the TZ* column of this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
         timeFormat               = "h:mm A"                # https://momentjs.com/docs/#/displaying/format/
+        timeUpdatePeriod         = 0                       # Update the localtime every timeUpdatePeriod milliseconds. This is usually either 0 (disabled) or 1000 (every second) 
```

Comment:
I don't think we need a config value for this.

Reply:
OK

Reply:
So you prefer a boolean ?